packages/picasso-forms/src/Form/story/index.jsx

```diff
@@ -263,3 +263,6 @@ however, you may need custom validators for more complex types of fields.
     description: 'Showcase how to use auto-save functionality.',
     takeScreenshot: false,
   })
+  .addExample('Form/story/HighlightAutofill.example.tsx', {
+    title: 'Highlight autofill',
```

Comment:
```suggestion
    title: 'Highlight fields with default value',
    description: 'Highlights fields with default value in yellow. Disappears on field being focused or modified.',
```

I would provide a bit more context on how it works so it's more clear from the storybook example

Reply:
✅